VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -1528,6 +1540,90 @@ end subroutine vom_write_dayyear
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+     subroutine vom_save_dayyear (output)
```

Comment:
Could you add a short comment what this subroutine does?

Reply:
I added a brief description